.travis.yml

```diff
@@ -10,7 +10,9 @@ services:
   - docker
 
 before_install:
+  - ./tools/travis/setupscan.sh
```

Comment:
it's not a big difference, but for consistency with the current repo WDYT if instead of `./tools/travis` folder we use `./.travis` ?